QgisModelBaker/libqgsprojectgen/dbconnector/pg_connector.py

```diff
@@ -412,6 +428,36 @@ def get_bags_of_info(self):
             return cur
         return []
 
+    def get_ignored_layers(self):
```

Comment:
This logic deserves to be in the parent class (DBConnector).

Reply:
I will do that.

Reply:
Done